testproject/Assets/Tests/Runtime/SceneLoadingTest.cs

```diff
@@ -1,3 +1,4 @@
+#if IGNORETHISTEST
```

Comment:
This is "OK".  [PR-1030](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/1030), the unit test for this PR, removes this completely.